src/Components/SideBar/SideBar.module.scss

```diff
@@ -0,0 +1,94 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;500&display=swap");
+// .............................................................
+
+$border-color: rgb(141, 141, 141, 0.2);
+
+* {
+  margin: 0;
+
+  font-family: poppins;
+}
+
+h4 {
+  width: 100%;
+  text-align: left;
+  padding: 2%;
+  font-family: poppins;
+}
+
+.mainBox {
+  float: right;
+  position: sticky;
+  top: 0;
+  background-color: white;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+
+  width: 35%;
+  height: 80vh;
+
+  text-align: center;
+}
+
+.box1 {
+  margin: 2rem auto 2rem auto;
```

Comment:
you can shorten this to:
margin: 2rem auto;

the first applies to top and bottom, while second for left and right.

Also, decide on a common unit. You use rem here, and vh above, and % for width as well (which is a modified version of vw which takes scrollbar into account).

Use this unit (vh-vw combo, or rem), and keep it same in as many places as possible (*not* all, but *as many as possible*)

Don't use pixel except in media query condition.